packages/typography/src/Typography.story.tsx

```diff
@@ -24,6 +33,23 @@ storiesOf('Typography', module).add('Default', () => (
       <InlineCode className={displayBlock}>InlineCode</InlineCode>
       <Disclaimer className={displayBlock}>Disclaimer</Disclaimer>
       <Overline>Overline</Overline>
+      <Link
+        href={select(
+          'href',
+          [
+            'http://localhost:9001',
+            'https://mongodb.github.io/leafygreen-ui/?path=/story/*',
+          ],
+          'http://localhost:9001',
+        )}
+        arrowAppearance={select(
+          'arrowAppearance',
+          ['hover', 'persistant' | 'none'],
```

Comment:
This probably should be `['hover', 'persistent', 'none']`. Right now the options in storybook are `'hover'` and `0` because of this.

Reply:
Typo, my bad! Thank you!